FodyAddinSamples.sln

```diff
@@ -35,108 +35,571 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WeakEventHandlerSample", "W
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ToStringSample", "ToStringSample\ToStringSample.csproj", "{AF91A2D4-58D3-4772-A41A-7EF0139CD0F2}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MimickSample", "MimickSample\MimickSample.csproj", "{C5BDEAA2-7DB7-45B1-B403-CB0FE7908130}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MimickSample", "MimickSample\MimickSample.csproj", "{C5BDEAA2-7DB7-45B1-B403-CB0FE7908130}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "InSolutionWeaving", "InSolutionWeaving\InSolutionWeaving.csproj", "{6E5EB9C2-705E-48F2-9A53-7957F33540B0}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "KasayDependencyPropertyUWPSample", "KasayDependencyPropertyUWPSample\KasayDependencyPropertyUWPSample.csproj", "{C078A0B9-1FA3-467C-985A-9B4D04A94839}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|ARM = Debug|ARM
+		Debug|ARM64 = Debug|ARM64
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
 		Release|Any CPU = Release|Any CPU
+		Release|ARM = Release|ARM
+		Release|ARM64 = Release|ARM64
+		Release|x64 = Release|x64
+		Release|x86 = Release|x86
 		ReleaseNetCore|Any CPU = ReleaseNetCore|Any CPU
+		ReleaseNetCore|ARM = ReleaseNetCore|ARM
+		ReleaseNetCore|ARM64 = ReleaseNetCore|ARM64
+		ReleaseNetCore|x64 = ReleaseNetCore|x64
```

Comment:
Can u exclude this on releasenetcore so it doesnt try to build on linux

Reply:
it was a sample in uwp, but I will simplify it

Reply:
i was referring to your project being included in the `ReleaseNetCore` build option. this will result in it be run on the linux build

Reply:
bump